checklists/alz_checklist.en.json

```diff
@@ -1978,7 +1980,8 @@
             "guid": "d5f345bf-97ab-41a7-819c-6104baa7d48c",
             "id": "F01.12",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/azure/azure-monitor/alerts/action-groups"
+            "link": "https://learn.microsoft.com/azure/azure-monitor/alerts/action-groups",
+            "training": "https://learn.microsoft.com/training/modules/intro-to-azure-service-health/"
```

Comment:
I feel this check is more about setting the action group for the alerts, something like this may be more suitable?
https://learn.microsoft.com/en-gb/training/modules/incident-response-with-alerting-on-azure/7-actions-and-alert-processing-rules

What are your thoughts?